core/tests/protractor_utils/editor.js

```
@@ -575,9 +579,10 @@ var ResponseEditor = function(responseNum) {
       element(by.css('.protractor-test-save-outcome-feedback')).click();
     },
     // This saves the rule after the destination is selected.
-    // Note that the supplied 'destinationName' must be an existing state. If
-    // 'createState' is true, it will attempt to create a state named after the
-    // input 'destinationName'.
+    //  - destinationName: The name of the state to move to, or null to stay on
+    //    the same state.
+    //  - createState: whether the destination state is new and must be created
+    //    at this point.
     setDestination: function(destinationName, createState) {
```

Maybe raise an error if destinationName is null and createState is true?